test/chain/block.cpp

```diff
@@ -262,30 +266,30 @@ BOOST_AUTO_TEST_CASE(block__from_data__insufficient_transaction_bytes__failure)
         "00000000000000000000000000000000000000000000ffffffff07049d8e2f1b"
         "0114ffffffff0100f2052a0100000043410437b36a7221bc977dce712728a954"));
 
-    chain::block instance;
+    chain::block instance(settings{});
     BOOST_REQUIRE(!instance.from_data(data));
     BOOST_REQUIRE(!instance.is_valid());
 }
 
 BOOST_AUTO_TEST_CASE(block__genesis__mainnet__valid_structure)
 {
-    const auto genesis = bc::chain::block::genesis_mainnet();
+    const auto genesis = bc::chain::block::genesis_mainnet(settings());
```

Comment:
Use consistent style (settings() and settings{}).

Reply:
My style is very consistent. I always use `settings()` except for cases where the vexing parse occurs.